saleor/plugins/webhook/tasks.py

```diff
@@ -71,7 +70,7 @@ def _get_webhooks_for_event(event_type, webhooks=None):
     if webhooks is None:
         webhooks = Webhook.objects.all()
 
-    webhooks = webhooks.filter(
+    webhooks = webhooks.order_by("app_id", "pk").filter(
```

Comment:
Just a comment here: we discussed with Mateusz that webhooks should be processed foremost in the order of app installation. Some webhooks (such as tax webhooks) return first relevant result rather than iterating over all potential plugins.

This means that, e.g. orders and checkouts could be processed by different plugins or that an app could "turn itself off" by replacing the existing webhook

Reply:
Is it tested? I think we should provide a test for that as this is something that might be easily broken by accident.